Userland/Applets/Audio/main.cpp

```diff
@@ -34,18 +35,18 @@ class AudioWidget final : public GUI::Widget {
 public:
     static ErrorOr<NonnullRefPtr<AudioWidget>> try_create()
     {
-        Vector<VolumeBitmapPair, 5> volume_level_bitmaps = {
+        auto volume_level_bitmaps = TRY(FixedArray<VolumeBitmapPair>::try_create{
             { 66, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-high.png")) },
             { 33, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-medium.png")) },
             { 1, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-low.png")) },
             { 0, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-zero.png")) },
             { 0, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-muted.png")) }
-        };
+        }));
         return adopt_nonnull_ref_or_enomem(new (nothrow) AudioWidget(move(volume_level_bitmaps)));
```

Comment:
This moves the objects from the stack to being on the heap, I'd suggest simply using an `Array<>` here instead as no one seems to be using it as an actual vector.

```suggestion
        return adopt_nonnull_ref_or_enomem(new (nothrow) AudioWidget({
            { 66, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-high.png")) },
            { 33, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-medium.png")) },
            { 1, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-low.png")) },
            { 0, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-zero.png")) },
            { 0, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/audio-volume-muted.png")) }
        }));
```

Reply:
I will still create a variable for this because it makes the code clearer to read than having one gigantic function call.

Reply:
how does creating a variable make it any clearer? the initializer is the same size with or without..

Reply:
Yes, but it's then not part of a new-expression which is part of a function call. That's a little too much nesting I think.